app/printer.py

```diff
@@ -0,0 +1,14 @@
+from app.book import Book
+
+
+class BookPrinter:
+    @staticmethod
+    def print_book(book: Book, print_type: str) -> None:
+        if print_type == "console":
+            print(f"Printing the book: {book.title}...")
+            print(book.content)
+        elif print_type == "reverse":
+            print(f"Printing the book in reverse: {book.title}...")
+            print(book.content[::-1])
+        else:
+            raise ValueError(f"Unknown print type: {print_type}")
```

Comment:
`else` redundant here 
```suggestion
        raise ValueError(f"Unknown print type: {print_type}")
```